staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/mutating/dispatcher.go

```diff
@@ -117,6 +118,9 @@ func (a *mutatingDispatcher) callAttrMutatingHook(ctx context.Context, invocatio
 		return &webhook.ErrCallingWebhook{WebhookName: h.Name, Reason: fmt.Errorf("webhook does not accept v1beta1 AdmissionReview")}
 	}
 
+	trace := utiltrace.New(fmt.Sprintf("mutating webhook request: %s", h.Name)) // TODO(jpbetz): use accessor.GetUID or similar here? Name is really not enough, although it is what we use for metrics.
```

Comment:
nit: searched other places, capitalizing the first letter seems to be our convention for trace messages